### PR TITLE
Support fonts in webpack

### LINKED ISSUE #195: Add support for fonts in webpack (woff, woff2, ttf and eot)
This looks like a common test regexp: `/\.(woff(2)?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/`.

We can try with a limit of 25kb.

### LINKED ISSUE #209: Global leaks accross multiple requests in SSR
**_Bug report_**

- [x] I've searched for existing issues
- [x] I've checked documentation: https://docs.frontity.org
- [x] I've checked community forum: https://community.frontity.org

**Describe the expected behavior**

Using different `<Global>` components for different URLs should include only the `<Global>` present in that URL.

**Describe the observed behavior**

Once you visit a URL, that `<Global>` component is always included, even tho it's not present in other URLs.

**Describe the steps involved to reproduce the problem**

Create two different `<Global>` components and include them in two different URL. Visit both URLs and look at the CSS inserted in the `<head>`. It contains both styles.

I've added an e2e test in this commit: https://github.com/frontity/frontity/commit/8649b8638b9ca43e0289574c1bcb76cee7616356

**Possible solutions**

I think it may be related to this line: 
https://github.com/frontity/frontity/blob/dev/packages/core/src/app/index.tsx#L2
Because we are using the same cache for all the SSR's.

I'd try creating a new cache with `@emotion/cache` inside the `<App>` component:
https://emotion.sh/docs/@emotion/cache

### PR DESCRIPTION
**Pull request by luisherranz**
#### Description of what you did:

Font support is already added but we need to fix the leaking of `<Global>` in order to make it work properly.

I've already added non-passing e2e tests for the Global component.

I listed the possible solutions in the issue: https://github.com/frontity/frontity/issues/209

Fixes https://github.com/frontity/frontity/issues/195
Fixes https://github.com/frontity/frontity/issues/209

#### My PR is a:

- 🐞 Bug fix
- 🔝 Improvement

#### Is the PR ready to be merged?

- ✅ Yes!

### CODE REVIEW

**Comment by luisherranz** [comment]
I solved the leak problem with `<Global>` and I included image snapshots in the e2e tests.

**Comment by luisherranz** [comment]
Hey @DAreRodz, I need this PR merged in order to finish the fix of emotion. Could please give it some priority? 

Thanks!! 😅🙏

**Review by DAreRodz — APPROVED** [review]
Everything looks great!

Just two things I commented, take a look at them.

File: e2e/packages/fonts/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+const All: React.FC = () => (
+  <>
+    <Global
+      styles={css`
+        @font-face {
+          font-family: "Aclonica";
+          font-style: normal;
+          font-weight: 400;
+          src: url(${aclonicaEot});
```
[the remark this answers is not in the document]
Comment (luisherranz) [reply e2e/packages/fonts/src/index.tsx:23]: Don't ask me why but it's the way it's done for Eot in old browsers. It is correct.

File: e2e/packages/fonts/fonts.d.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+declare module "*.woff";
```
[the remark this answers is not in the document]
Comment (luisherranz) [reply e2e/packages/fonts/fonts.d.ts]: You're right. I guess this was done before we added `type-declarations`.

File: e2e/packages/fonts/src/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (DAreRodz) [reply e2e/packages/fonts/src/index.tsx:23]: 👍

**Pull request merged**